pkg/manifest/v1/indicator.go

```diff
@@ -40,7 +40,7 @@ type SLI struct {
 
 // SLIInline represents the SLI inline.
 type SLIInline struct {
-	Metadata Metadata `yaml:"metadata" validate:"required"`
+	Metadata Metadata `yaml:"metadata,omitempty" validate:"omitempty"`
```

Comment:
is this intentional change? If so, please revert this, `metadata` is a required field for inline definitions